kratos/factories/register_explicit_builder_factory.cpp

```diff
@@ -0,0 +1,41 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "factories/base_factory.h"
+#include "spaces/ublas_space.h"
+
+// Builder And Solver
+#include "solving_strategies/builder_and_solvers/explicit_builder.h"
+
+namespace Kratos
+{
+void RegisterExplicitBuilders()
```

Comment:
this is not consistent with the file name

Reply:
True